src/main/java/com/jadajabajava/services/EmployerService.java

```diff
@@ -0,0 +1,30 @@
+package com.jadajabajava.services;
+
+import com.jadajabajava.entities.Employer;
+import com.jadajabajava.repositories.EmployerRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Slf4j
+@Service
+public class EmployerService extends AbstractService<Employer> {
+
+    private final EmployerRepository employerRepository;
+
+    private final MessageSource messageSource;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'messageSource' could be replaced by a local variable.](https://app.codacy.com/gh/jada-jaba-java/job-seeker-service/pullRequest?prid=5707872)